applications/MultilevelMonteCarloLaplacianApplication/python_scripts/multilevel_montecarlo_laplacian_analysis.py

```diff
@@ -0,0 +1,724 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as Kratos
+# KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING) # avoid printing of Kratos things
+Kratos.Logger.GetDefaultOutput().SetSeverity(Kratos.Logger.Severity.WARNING)
+import KratosMultiphysics.MyStochasticLaplacianApplication as Poisson
+
+from analysis_stage import AnalysisStage
+
+import numpy as np
+import time
+
+from pycompss.api.task import task
```

Comment:
throw a meaningful error if pycompss is not installed

Reply:
I commented the lines where I import pycompss and the lines where I define the task, and I get no error. Try if this works.